### PR TITLE
dualstack: add periodic for 1.19, 1.20

### PR DESCRIPTION
**Pull request by aramase**
- Removes periodic tests for unsupported releases: `1.16` and `1.17`
- Adds periodic tests for `1.19` and `1.20`
- Enables conformance testing for `1.20` branch

### CODE REVIEW

**Comment by aramase** [comment]
cc @lachie83

File: config/jobs/kubernetes/sig-network/dualstack-e2e.yaml [hunk 1]
```diff
@@ -143,15 +143,15 @@ periodics:
       - --aksengine-template-url=https://raw.githubusercontent.com/aramase/manifests/main/dualstack/ipvs-kubenet.json
```
Comment (aojea) [inline config/jobs/kubernetes/sig-network/dualstack-e2e.yaml:143]: just curious, why is the template name`ipvs-kubenet` for both iptables and ipvs jobs?

Comment (aramase) [reply config/jobs/kubernetes/sig-network/dualstack-e2e.yaml:143]: `1.18`, `1.19` and `1.20` are running with `ipvs-kubenet` templates. The master CI jobs have `iptables` and `ipvs`.

Comment (aojea) [reply config/jobs/kubernetes/sig-network/dualstack-e2e.yaml:143]: ahh, right, the github diff was collapsing some lines and I got confused

**Pull request merged**